src/go/rpk/pkg/cli/cmd/cloud/namespaces.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cloud
+
+import (
+	"fmt"
+	"io"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/vcloud/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/vcloud/ui"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/vcloud/yak"
+)
+
+func NewNamespacesCommand(fs afero.Fs, out io.Writer) *cobra.Command {
+	return &cobra.Command{
+		Use:   "namespaces",
+		Short: "Get namespaces in your vectorized cloud",
+		Long:  `List namespaces that you have created in your vectorized cloud organization.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			yakClient := yak.NewYakClient(config.NewVCloudConfigReaderWriter(fs))
+			return GetNamespaces(yakClient, out)
+		},
+	}
+}
+
+func GetNamespaces(c yak.CloudApiClient, out io.Writer) error {
+	ns, err := c.GetNamespaces()
+	if _, ok := err.(yak.ErrLoginTokenMissing); ok {
+		log.Info("Please run `rpk cloud login` first. ")
+		return err
+	}
+	if err != nil {
+		return err
+	}
+
+	printFormatted(ns, out)
+	return nil
+}
+
+func printFormatted(ns []*yak.Namespace, out io.Writer) {
```

Comment:
Instead of passing the `io.Writer` all the way through the command chain, you can use `logrus.StandardLogger.out`, which can be replaced like you've already done in the tests: https://github.com/vectorizedio/redpanda/pull/1322/commits/955ffcde6068eee2339d14ea87631eea40270a9b#diff-6e3421a317be37fe178dc2760515ebcc9cdf8e15b4c972c00882ec7788e9a994R37

Reply:
@alenkacz does this make sense?